Core/interface/RootExt.h

```diff
@@ -12,6 +12,8 @@ This file is part of https://github.com/hh-italian-group/TauMLTools. */
 #include <TMatrixD.h>
 #include <TFile.h>
 #include <Compression.h>
+#include <TAxis.h>
```

Comment:
Including TAxis is not necessary in RootExt.h. If needed, include it in RootExt.cpp.